README.md

```diff
@@ -394,6 +394,42 @@ type Output struct {
 }
 ```
 
+### Error Wrapping
+
+With `goverter:wrapErrors`, goverter will wrap conversion errors with additional troubleshooting information.
+The tag `goverter:wrapErrors` does not take parameters. The `wrapErrors` flag can also be set via command line.
```

Comment:
Two sentences begin with "The", what do you think of this?

```
You can enable error wrapping with the `-wrapErrors` CLI flag or by adding
`// goverter:wrapErrors` to the method or converter. Both parameters don't take any arguments.

Wrapped errors will contain troubleshooting information like the path to the
field that produced an error.
```

Reply:
I changed as recommended, with the exception of the path (I have not implemented it so I left that part out).